src/test/java/entities/MessageTest.java

```diff
@@ -0,0 +1,46 @@
+package entities;
+
+import org.junit.*;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import static org.junit.Assert.*;
+
+ /**
+  * A class that the test the Message class
+  */
+ public class MessageTest {
```

Comment:
Please add Javadoc tags for test cases- just briefly explaining what is being tested.